roles/openshift_node/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+# tasks file for openshift_node
+- name: Install OpenShift
+  yum: pkg=openshift-node state=installed
+
+  # fixme: Once openshift stops resolving hostnames for node queries remove this...
+- name: Set hostname to IP Addr (WORKAROUND)
+  command: /usr/bin/hostname {{ oo_bind_ip }}
+
+- name: Retrieve OpenShift Master credentials
+  local_action: command /usr/bin/rsync --compress --archive --rsh 'ssh  -S none -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null' root@{{ oo_master_public_ips[0] }}:/var/lib/openshift/openshift.local.certificates/admin/ /tmp/openshift
```

Comment:
It would be nice if we could do this just one time instead of doing it each time the nodes are configured.

Maybe add a TODO to figure out how we can do that. Right not it's not possible because of cluster.sh running the config twice. But eventually it should be able to be done.